topi/python/topi/cuda/group_conv2d_nchw.py

```diff
@@ -99,6 +102,17 @@ def group_conv2d_nchw_cuda(cfg, data, kernel, stride, padding, dilation, groups,
     oc_chunk, _, kernel_h, kernel_w, oc_block, ic_block = get_const_tuple(
         packed_kernel.shape)
 
+    # TODO(kumasento): these assertions ensure that the number of groups
+    # should be larger or equal to the number of blocks, so that each
+    # group will have at least one block.
+    # Shall we pad the channels to avoid raising assertions?
+    assert groups >= oc_chunk, \
```

Comment:
@kumasento should be less than
`assert groups <= oc_chunk`

Reply:
Ugh silly me! Thanks for pointing that out!